runtime/lua/vim/filetype.lua

```diff
@@ -1372,6 +1370,10 @@ local pattern = {
       return "git"
     end
   end,
+  [".*/etc/gitconfig"] = "gitconfig",
+  [".*/.config/git/config"] = "gitconfig",
```

Comment:
`.` here needs to be escaped

```suggestion
  [".*/%.config/git/config"] = "gitconfig",
```

Reply:
D'oh!

Reply:
Turns out that was already part of the original `filetype.lua`...

(We do need to come up with a way of sorting these...)